locale/en/foreman_remote_execution.po

```diff
@@ -1,19 +1,760 @@
+msgstr ""
+
+msgid "Provider type"
+msgstr ""
+
+msgid "Whether or not the template is locked for editing"
+msgstr ""
+
+msgid "Effective user options"
+msgstr ""
+
+msgid "What user should be used to run the script (using sudo-like mechanisms)"
+msgstr ""
+
+msgid ""
+"Whether it should be allowed to override the effective user from the invocatio"
+"n form."
+msgstr ""
+
+msgid "Whether the current user login should be used as the effective user"
+msgstr ""
+
+msgid "Create a job template"
+msgstr ""
+
+msgid "Update a job template"
+msgstr ""
+
+msgid "Template version"
+msgstr ""
+
+msgid "Delete a job template"
+msgstr ""
+
+msgid "Clone a provision template"
+msgstr ""
+
+msgid "List template inputs"
+msgstr ""
+
+msgid "Show template input details"
+msgstr ""
+
+msgid "Input name"
+msgstr ""
+
+msgid "Input description"
+msgstr ""
+
+msgid "Input is required"
+msgstr ""
+
+msgid "Input is advanced"
+msgstr ""
+
+msgid "Input type"
+msgstr ""
+
+msgid "Fact name, used when input type is fact"
+msgstr ""
+
+msgid "Variable name, used when input type is variable"
+msgstr ""
+
+msgid "Puppet class name, used when input type is puppet_parameter"
+msgstr ""
+
+msgid "Puppet parameter name, used when input type is puppet_parameter"
+msgstr ""
+
+msgid "Selectable values for user inputs"
+msgstr ""
+
+msgid "Create a template input"
+msgstr ""
+
+msgid "Delete a template input"
+msgstr ""
+
+msgid "Update a template input"
+msgstr ""
+
+msgid ", note that you must save template input changes before you try to preview it."
+msgstr ""
+
+msgid "Run Job"
+msgstr ""
+
+msgid "Run"
+msgstr ""
+
+msgid "Success"
+msgstr ""
+
+msgid "Failed"
+msgstr ""
+
+msgid "Pending"
+msgstr ""
+
+msgid "Cancelled"
+msgstr ""
+
+msgid "queued"
+msgstr ""
+
+msgid "running %s%"
+msgstr ""
+
+msgid "succeeded"
+msgstr ""
+
+msgid "failed"
+msgstr ""
+
+msgid "unknown status"
+msgstr ""
+
+msgid "cancelled"
+msgstr ""
+
+msgid "success"
+msgstr ""
+
+msgid "pending"
+msgstr ""
+
+msgid "Host detail"
+msgstr ""
+
+msgid "Rerun on %s"
+msgstr ""
+
+msgid "N/A"
+msgstr ""
+
+msgid "Refresh"
+msgstr ""
+
+msgid "Refresh this page"
+msgstr ""
+
+msgid "Rerun"
+msgstr ""
+
+msgid "Rerun the job"
+msgstr ""
+
+msgid "Rerun failed"
+msgstr ""
+
+msgid "Rerun on failed hosts"
+msgstr ""
+
+msgid "Job Task"
+msgstr ""
+
+msgid "See the last task details"
+msgstr ""
+
+msgid "Cancel Job"
+msgstr ""
+
+msgid "Try to cancel the job"
+msgstr ""
+
+msgid "Task Details"
+msgstr ""
+
+msgid "See the task details"
+msgstr ""
+
+msgid "Try to cancel the job on a host"
+msgstr ""
+
+msgid "in %s"
+msgstr ""
+
+msgid "%s ago"
+msgstr ""
+
+msgid "Documentation"
+msgstr ""
+
+msgid "Template input"
+msgstr ""
+
+msgid "remove template input"
+msgstr ""
+
+msgid "Use default description template"
+msgstr ""
+
+msgid "Description template"
+msgstr ""
+
+msgid "Explanation"
+msgstr ""
+
+msgid ""
+"This template is used to generate the description.\n"
```

Comment:
Whitespace is a bit of a mess here, you might need to use heredoc or change how it's represented in the source.